index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <link rel="stylesheet" type="text/css" href="style.css" />
+    </head>
+
+    <body class="container">
+        <div class="header">
+            <div class="logo">
+                <img src="My Logo.png" alt="" class="logo-img" />
+            </div>
+            <div class="logo"></div>
+        </div>
+
+        <div class="first-div">I'm Alexander. Glad to see you</div>
+        <div class="second-div">
+            i'm a software developer! i can help you build a product, feature or
+            website look through some of my work and experience! if you like
+            what you see and have a project you need coded, dont hesitate to
+            contact me
+        </div>
+        <div class="connect">LET'S CONNECT</div>
+        <div class="icon-div">
+            <img src="Vector (2).svg" alt="" />
+            <img src="Linkedin icon.svg" alt="" />
+            <img src="Vector (3).svg" alt="" />
+            <img src="Vector (4).svg" alt="" />
+            <img src="Vector (5).svg" alt="" />
+        </div>
```

Comment:
- [ ] Kindly wrap this part into a `section` tag. That would be semantically correct.